algorithms/gfg_arrays1.py

```diff
@@ -1,7 +1,13 @@
-from merge_sort import merge_sort
+from algorithms.merge_sort import merge_sort
 
 
-def is_there_sum_of_two_elements(keys, keys_length, v):
+def is_there_sum_of_two_elements(keys, v):
+    """
+    Write a program that, given an array of n numbers and another number x, 
```

Comment:
A nit really: I would take away that "write a program that" (because this function doesn't write a program)